deepmd/pt/model/task/denoise.py

```
@@ -83,55 +307,309 @@
+            "use_aparam_as_mask": self.use_aparam_as_mask,
+            "spin": None,
+        }
+
+    def deserialize(cls, data: dict) -> "DenoiseNet":
+        data = data.copy()
+        variables = data.pop("@variables")
+        cell_nets = data.pop("cell_nets")
+        coord_nets = data.pop("coord_nets")
+        token_nets = data.pop("token_nets")
+        obj = cls(**data)
+        for kk in variables.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)